rhcos4/profiles/cis.profile

```diff
@@ -0,0 +1,63 @@
+documentation_complete: true
+
+title: 'CIS Red Hat OpenShift Container Platform 4 Benchmark'
+
+description: |-
```

Comment:
I think that the description should mention that this is going to be thrown away.

Reply:
Thank you for taking a look, in general I agree but with @yuumasato 's help we've been able to fix the cis-node profile, so I'm going to close this PR and continue there.